Formula/hh-suite.rb

```diff
@@ -4,7 +4,7 @@ class HhSuite < Formula
   homepage "https://github.com/soedinglab/hh-suite"
   url "https://github.com/soedinglab/hh-suite/archive/v3.2.0.tar.gz"
   sha256 "6b870dcfbc1ffb9dd664a45415fcd13cf5970f49d1c7b824160c260fa138e6d6"
-  revision 1
+  revision 2
```

Comment:
Since these changes are aesthetic and don't affect the resulting bottle (binary package), there's no need to bump the revision.